plugins/woocommerce/src/Internal/DataStores/Orders/DataSynchronizer.php

```diff
@@ -89,6 +101,8 @@ public function __construct() {
 		self::add_action( 'woocommerce_refund_created', array( $this, 'handle_updated_order' ), 100 );
 		self::add_action( 'woocommerce_update_order', array( $this, 'handle_updated_order' ), 100 );
 		self::add_action( 'wp_scheduled_auto_draft_delete', array( $this, 'delete_auto_draft_orders' ), 9 );
+		self::add_action( 'init', array( $this, 'handle_background_sync' ) );
```

Comment:
`init` is noisy, should we switch to admin_init instead?

Reply:
I explained my reasoning [here](https://github.com/woocommerce/woocommerce/pull/39952#discussion_r1327735839), for sites where there isn't a user consistently logging into WP Admin, I don't think the callback would fire frequently enough.

Reply:
Switched to the `shutdown` hook in 9c51363d87167de1b4762e7f64580a339ce6449f